readme.md

```diff
@@ -632,6 +632,7 @@
 
 ## Security
 
+- [Annual Security Reports](https://github.com/jacobdjwilson/awesome-annual-security-reports#readme) - Annual cybersecurity analysis and industry survey reports
```

Comment:
Should be added at the bottom.

Reply:
> Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.**

Reply:
I have moved the entry to the bottom of the list and added a meaningful description